### DISCUSSION TITLE
Webinar: Akka.Cluster Simply Explained - June 30th, 2021, 10am CST

### DISCUSSION THREAD

**Discussion by Aaronontheweb** [akkadotnet/akka.net#5101] [Show and tell]
# Akka.Cluster Simply Explained
June 30th, 2021, 10am CST 

**Register** https://us06web.zoom.us/webinar/register/WN_W2bhHt3hQvO2FeC2aTAsdA?goal=0_cb6592d396-2aed6c8029-93394713

## Abstract
In this webinar we will cover all of the questions you might have about Akka.NET and running Akka.Cluster:

1. What is it for?
2. Why and when should I use it?
3. How does it work?
4. How do I deploy it?
5. What else do I need to know about it?

We will explore some of the more popular modules of Akka.Cluster, such as:

1. Akka.Cluster routers;
2. Akka.Cluster singletons;
3. Akka.Cluster.Sharding and how entity actors are created / moved;
4. Distributed publish-subscribe and how to use it; and
5. Akka.Cluster.DistributedData for replication and recovery.

This webinar should be 90 minutes long and we will make time to take questions. A recording of this webinar will be posted to [Petabridge's YouTube channel](https://www.youtube.com/c/PetabridgeAcademy).

**Comment by go-aegian** [comment]
Thank you Aaron, definitely a way to learn for me.

**Reply by Aaronontheweb** [reply]
Happy to do it.
